### DISCUSSION TITLE
Changing theme of Most Used Languages

### DISCUSSION THREAD

**Discussion by Safuan12616** [anuraghazra/github-readme-stats#1329] [Q&A] [answered]
### Changing theme of Most Used Languages.
You can also change themes of **Most Used Languages**.
You just need to change the `theme name`
`&layout=compact&theme=theme name`
Preview:
![image](https://user-images.githubusercontent.com/80021390/134848253-8f3b2099-7650-45a4-8ee0-0371f127733b.png)
Here I used `discord_old_burple` theme.

**Comment by anuraghazra** [comment, marked as answer]
Yeah it's documented: https://github.com/anuraghazra/github-readme-stats#common-options

**Comment by DevX-Cipher** [comment]
I have noticed that it's not even if you look here the yellow ones for Pascal are higher then the others and it's a noticeable difference
![image](https://github.com/anuraghazra/github-readme-stats/assets/25472227/83308735-a94f-42a5-b1b4-aa2955c1391a)

**Reply by qwerty541** [reply]
Hey, @modz2014! I'm not sure I understood you correctly, but if you're talking about languages colors, then we don't define them ourselves. We load them from https://github.com/github-linguist/linguist using special workflow [update-langs.yaml](https://github.com/anuraghazra/github-readme-stats/blob/master/.github/workflows/update-langs.yaml). It means that our card shows same languages color that GitHub interface.

**Comment by DevX-Cipher** [comment]
Upon careful examination of the image, it becomes evident that the elements do not align in a cohesive manner, resulting in an overall appearance of irregularity and imbalance.

**Reply by qwerty541** [reply]
I suppose that you're talking about different length of strings with programming languages names and percentages. If you don't like this layout feel free to propose the better one. We already provide [several different layouts](https://github.com/anuraghazra/github-readme-stats#demo-2) for top languages card.

**Comment by DevX-Cipher** [comment]
I like this one if you look closer in the details you will notice but it
does not show the defect when your on a phone which is werid

**Reply by qwerty541** [reply]
It would be great if you could attach a screenshot of the defect and circle it for clarity, since apparently all this time we are talking about different things.

**Reply by DevX-Cipher** [reply]
i did above mate

**Reply by qwerty541** [reply]
I see this screenshot https://github.com/anuraghazra/github-readme-stats/discussions/1329#discussioncomment-7467260 but i don't see any defect on it, so i asked you to circle it.

**Comment by DevX-Cipher** [comment]
itrs only small detials but it works fine when i view it on a mobile phone 
![280293820-83308735-a94f-42a5-b1b4-aa2955c1391a](https://github.com/anuraghazra/github-readme-stats/assets/25472227/bcae2724-b270-478f-b0b3-6e6674257d33)

**Reply by qwerty541** [reply]
Yes, I see this little piece where two colors overlap each other. I checked the code which generates compact layout of top languages card and noticed that each next part of progress bar starts from same coordinates where previous ends, without any indent. https://github.com/anuraghazra/github-readme-stats/blob/master/src/cards/top-languages-card.js#L360-L382

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/13b955fb-7b50-40e6-813c-8b60dac0d902)

In general, this is not critical, but of course it can be fixed, but you will need to very carefully select the size of the indentation so that the black defect does not change to white. 

@rickstaa What do you think? Is it worth to change this in your opinion?

**Reply by DevX-Cipher** [reply]
would make it look nicer though

**Reply by rickstaa** [reply]
> Yes, I see this little piece where two colors overlap each other. I checked the code which generates compact layout of top languages card and noticed that each next part of progress bar starts from same coordinates where previous ends, without any indent. https://github.com/anuraghazra/github-readme-stats/blob/master/src/cards/top-languages-card.js#L360-L382
> 
> ![image](https://user-images.githubusercontent.com/53787217/282234384-13b955fb-7b50-40e6-813c-8b60dac0d902.png)
> 
> In general, this is not critical, but of course it can be fixed, but you will need to very carefully select the size of the indentation so that the black defect does not change to white.
> 
> @rickstaa What do you think? Is it worth to change this in your opinion?

Since this is a minor bug, it should be low on our priority list. I didn't notice it until you brought it to my attention 😅 . Having that said, maybe @modz2014, you can [create a bug report](https://github.com/anuraghazra/github-readme-stats/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) for this, and if people want, they can then create a pull request we can review 👍🏻.

**Reply by qwerty541** [reply]
I would like to solve this problem personally, but unfortunately it is not reproducible on any of my devices. Apparently its appearance also depends on the screen resolution, or something similar. This behavior once again casts doubt on making any fix for this at all.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/2a1d7a27-06c4-433b-8c0c-bfb1d79cb133)

**Comment by DevX-Cipher** [comment]
So it works fine on my phone i see must but the screen i use seems werid

**Reply by qwerty541** [reply]
Let's keep this issue https://github.com/anuraghazra/github-readme-stats/issues/3469 open and track community demand on this problem. If it can be reproduced only on old devices and affects on very low percentage of our users, there is no sense in doing something about it in my opinion. If there will be a lot of people who experience same problem we can fix it. What do you think @rickstaa?

**Reply by rickstaa** [reply]
Hey @qwerty541, I'm on board with the idea of tracking this issue, but I reckon #3469 could use a bit of a makeover. A clear example, some visuals, and a more polished problem description would work wonders, don't you think?

**Reply by qwerty541** [reply]
@rickstaa Yes, I completely agree with you that this issue needs to be redone. Let's give @modz2014 ability to make the suggested changes since this issue was opened by him. Of course we can provide him help later if he needs.

**Reply by DevX-Cipher** [reply]
i have to look into more but im sure you guys know :)

**Reply by qwerty541** [reply]
@rickstaa I have updated this issue https://github.com/anuraghazra/github-readme-stats/issues/3469 and added my comment. Please share your minds about if it become clear now or not.

**Reply by rickstaa** [reply]
@qwerty541 ah, great! Thanks a lot for putting in the effort to update the issue 🚀. Let's use the issue to see how many people are affected 👍🏻.
